app/vmalert/notifier/notifier_blackhole.go

```diff
@@ -0,0 +1,35 @@
+package notifier
+
+import "context"
+
+// BlackHoleNotifier is can be used when no notifications needs to be sent
```

Comment:
// BlackHoleNotifier is used when no notification needs to be sent

Reply:
made changes